NBitcoin/TransactionBuilder.cs

```diff
@@ -664,7 +687,21 @@ public TransactionBuilder SetLockTime(LockTime lockTime)
 			return this;
 		}
 
+		Sequence? _Sequence;
+		public TransactionBuilder SetRelativeLockTime(int lockHeight)

```

Comment:
I think you can just use `SetSequence(Sequence)`.

My only problem with this is that sequence is a per input concept, not a global tx concept. 
But I guess this is still useful if a wallet has several keys depending on the same miniscript?

Reply:
I think the api to take `int blockHeight` or `TimeSpan period` is better than taking Sequence because we must assure tx nVersion is bigger than 1.
users are not interested into how the rule is encoded. they just want to use the relative locktime feature.

What about
`SetRelativeLockTimeTo(ICoin whichCoin, int lockHeight)`

and hold the info as `Dictionary<ICoin, Sequence>` ? In this way, we can assure that it will affect only specific input.

Reply:
Tell me if the current implementation is not good. I decided to use `Dictionary<OutPoint, Sequence>` since Coins does not support `GetHashCode`